web/webhook_test.go

```diff
@@ -263,12 +261,10 @@ func TestCommandWebhooks(t *testing.T) {
 	assert.Equal(t, http.StatusBadRequest, resp.StatusCode)
 
 	for i := 0; i < 5; i++ {
-		if resp, appErr := http.Post(ApiClient.Url+"/hooks/commands/"+hook.Id, "application/json", bytes.NewBufferString(`{"text":"this is a test"}`)); err != nil || resp.StatusCode != http.StatusOK {
-			t.Fatal(appErr)
-		}
+		response, appErr2 := http.Post(ApiClient.Url+"/hooks/commands/"+hook.Id, "application/json", bytes.NewBufferString(`{"text":"this is a test"}`))
+		require.False(t, appErr2 != nil || response.StatusCode != http.StatusOK)
```

Comment:
Let's split these up and assert one as `NotNil` and another as an `Equals`, so that in the event the test catches a regression, we know precisely why.